src/index.js

```diff
@@ -0,0 +1,15 @@
+import React from 'react';
+import ReactDOM from 'react-dom/client';
+import { Provider } from 'react-redux';
+import store from './redux/store';
+import './index.css';
+import App from './App';
+
+const root = ReactDOM.createRoot(document.getElementById('root'));
+root.render(
+  <React.StrictMode>
+    <Provider store={store}>
+      <App />
+    </Provider>
+  </React.StrictMode>,
```

Comment:
- [ ] All Redux actions should be dispatched once, So please remove the `strict mode` here because pending and fulfillment actions of the rockets and dragons are dispatched twice.

![image](https://github.com/BHObserver/Space-Travelers-Hub/assets/68796232/fddc9352-600f-4b0a-948c-de9d2184f480)